openvpn-access-server-ubuntu/azuredeploy.json

```diff
@@ -28,26 +28,26 @@
     },
 	"vmSize": {
       "type": "string",
-      "defaultValue": "Standard_A1",
+      "defaultValue": "Basic_A0",
       "allowedValues": [
-	    "Standard_A1",
-		"Standard_A2",
-		"Standard_A3",
-        "Standard_D1",
-        "Standard_D3",
-		"Standard_D4"
+            "Basic_A0",
```

Comment:
remove allowed values as it will prevent deployment in regions that don't support those